### ISSUE TITLE
OCR with text indexing

### ISSUE THREAD

**Issue by gggirlgeek** [clipto-pro/Android#80]
It would amazing if Clipto could allow us to search for text within images like Evernote does (now only in paid version.)

**Comment by jstetten** [comment]
I've started using Clipto more and more as an Evernote type replacement and have to say I wish for this nearly every day. Does Google make its Lens API available to developers now? I know there are several other ones, too.

Of equal, or even greater, usefulness would be automatic text transcriptions of voice recordings: either shared from the voicemail app or even dictated straight into a note in a future version. How cool would that be? 

Whatever is chosen eventually, I'd prefer (hopefully @atrashler agrees) to err on the side of privacy and trustworthiness than perfect accuracy. (Or make the choice of service an option?)

**Comment by gggirlgeek** [comment]
Yeah, voice transcribing would be neat but now we're talking about services that almost certainly need to be hosted, and that requires membership and privacy worries. Not sure I'd be happy about ClipTo going there. I ended up here trying to escape Evernote. You know?

I'd love to know how you use Clip To for note taking. I'm still in search of a good app for that. I'll try to start a discussion here for people to share how they use it -- maybe a discussion about notes exclusively. I'll start something after I hunt around for other discussions.....

**Comment by jstetten** [comment]
@gggirlgeek, please do start a discussion, I'd like to know how other power users are using Clipto also.

As wonderfully feature-filled and visually appealing as the app is, I'm still running up against missing features that are important _for me_ to be a complete replacement for all my various productivity apps. Chief among them is the lack of true to-do checklists as built-in first-class entities...ie., manually sortable, draggable, groupable, indented items that can be easily checked (and hidden) or un-checked. For this one use case, Google Keep knocks it out of the park. Doesn't hurt that it has built-in reminders too, my second request. 3rd would be the display of thumbnails of attached images in lists, and medium-sized images/previews that display _inline_ as part of the note when clicked, like Keep. If Keep played better with Clipto (or vice-versa), when it came to exporting checklists, I'd be happy to use both. If there was some way to link to 🔗 a live Keep note inside Clipto, that would be ideal. In fact, there are all sorts of entities from other apps I would _like_ to link to someday as living, editable objects when attaching a static screenshot/export of said live item (a family grocery/errand checklist, a spreadsheet of hourly weather records, a screenplay undergoing revisions) would be quickly out-of-date and makes no sense. 

I've been planning to open up some feature request issues, but I also don't want to overload the one (two?) developer(s), who seem extremely busy as it is. Also, I do get it: it is a clipboard manager at heart 💛...but even @atrashler has said he has plans to expand Clipto far beyond what its humble name implies. Given it's already expansive, impressive capabilities as it is, I don't think I'm totally crazy bringing up these ideas. But I've already written far more than I planned to here. @gggirlgeek, where is that discussion? 😉
